### PR TITLE
feat(SectionMessage): initial implementation

### LINKED ISSUE #151: Section Message
### Variations:
**Types**: Info, Success, Warning, Error

**Design reference:**
[Figma file - Section message](https://www.figma.com/file/y6Ca8BdJtQEd3Of1saRieF/Spark-Design-System-Web?node-id=2379%3A4207)

### PR DESCRIPTION
**Pull request by WilliamKelley**
Builds off of #256 

Closes #151

### CODE REVIEW

**Review by seigler — APPROVED** [review]

**Comment by WilliamKelley** [comment]
@slingner @seigler review bump

**Comment by WilliamKelley** [comment]
@slingner @shadoath review bump

File: libs/spark/stories/section-message.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    href: 'https://v4.mui.com/components/alert/#alert',
+  },
+  props: {
+    extends: {
+      href: 'https://v4.mui.com/api/alert/#props',
+    },
+    omits: [
+      {
+        name: 'variant',
+        defaultValue: "'standard'",
```
Comment (shadoath) [inline libs/spark/stories/section-message.stories.tsx:89]: Why does this need the double quoting? Is it a string referencing the value which is also a string?

**Review by shadoath — APPROVED** [review]

File: libs/spark/stories/section-message.stories.tsx (hunk 1 above)
Comment (WilliamKelley) [reply libs/spark/stories/section-message.stories.tsx:89]: yes

**Pull request merged**